wl2/src/Components/Main.js

```diff
@@ -1,39 +1,56 @@
-import React from "react";
+import React, { useState } from "react";
 import { NavLink } from "react-router-dom";
 import {
   // ScheduledItems
-  SearchButton
-  ,CreateList

```

Comment:
FYI the reason I put the comma at the beginning is that it's easier to selectively disable individual components if desired/needed by simply commenting them out. If the comma is at the end of the prior component, it takes far more efforts to disable individual lines. ;)